static/js/containers/pages/CheckoutPage_test.js

```diff
@@ -89,6 +89,23 @@ describe("CheckoutPage", () => {
     assert.equal(inner.find("CheckoutForm").prop("couponCode"), code)
   })
 
+  it("parses the preselect ID from the query parameter and verifies it exists in selected runs", async () => {
+    const preselect = "1234"
+    const { inner } = await renderPage(
+      {},
+      {
+        location: {
+          search: `product=4567&preselect=${preselect}`
+        }
+      }
+    )
+    // There is no run with the specified ID so it will pop up against undefined key
+    assert.equal(
+      inner.find("CheckoutForm").prop("selectedRuns")[undefined],
```

Comment:
We are not testing what this test is supposed to do. We should test a scenario of actual selectedRuns.

Reply:
Modified the test to use expected value for courseId as well.